process_report/process_report.py

```diff
@@ -62,6 +70,23 @@ def is_old_pi(old_pi_dict, pi, invoice_month):
     return False
 
 
+def get_invoice_bucket():
+    b2_resource = boto3.resource(
+        service_name="s3",
+        endpoint_url=os.environ["B2_ENDPOINT"],
+        aws_access_key_id=os.environ["B2_KEY_ID"],
+        aws_secret_access_key=os.environ["B2_APP_KEY"],
```

Comment:
If any of those environment variables are not set then it'll raise a key error, you could handle that cleanly and say that one of those were not set.

For endpoint_url, if you use os.environ.get then you could set a default value
```python
  endpoint_url=os.environ.get("B2_ENDPOINT", "https://s3.us-east-005.backblazeb2.com")
```